plugins/olm/commands/diagnose.md

```
@@ -0,0 +1,410 @@
+---
+description: Diagnose and optionally fix common OLM and operator issues
+argument-hint: [operator-name] [namespace] [--fix] [--cluster]
+---
+
+## Name
+olm:diagnose
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)